src/windows/functions.cpp

```diff
@@ -393,6 +393,38 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   return MOUNTUTILS_SUCCESS;
 }
 
+BOOL IsDriveEjectable(TCHAR driveLetter) {
+  TCHAR devicePath[8];
+  wsprintf(devicePath, TEXT("%c:\\"), driveLetter);
```

Comment:
https://msdn.microsoft.com/en-us/library/windows/desktop/ms647550(v=vs.85).aspx warns that `wsprintf` shouldn't be used?